v2/recipes/state-machine-transitions.md

```diff
@@ -0,0 +1,73 @@
+# State Machine Transitions
+
+For an actual example being used in Drupal Commerce, check out the state machine transition event subscriber being used in commerce_order to set an order's placed timestamp:
+* [commerce_order.services.yml](https://github.com/drupalcommerce/commerce/blob/080ca52fbb9ec73b9eeece5487a62d221e75ed04/modules/order/commerce_order.services.yml#L29)
+* [TimestampEventSubscriber.php](https://github.com/drupalcommerce/commerce/blob/080ca52fbb9ec73b9eeece5487a62d221e75ed04/modules/order/src/EventSubscriber/TimestampEventSubscriber.php)
+
+## Finding Transitions
+Transition information can be found in a `{module}.workflows.yml`.
```

Comment:
"Each order type can have its own transitions. You will need to check which workflow your order type is using. By the default order type uses the `order_default` workflow."

"All workflows, however, are required to have a `place` and `cancel` transiton."